index.js

```diff
@@ -71,14 +87,14 @@ ErrorCat.prototype.respond = function (err, req, res, next) {
     err.isBoom ? err.output.payload : 'Internal Server Error'
   ));
 };
+/* jslint unused:true */
 
 /**
- * Logs errors via debug, and reports them to rollbar.
+ * Logs errors via debug
  * @param {Error} err The error to log.
  */
 ErrorCat.prototype.log = function (err) {
   this.debug(err);
-  this.report(err);
 };
```

Comment:
thought about that... I'm okay with the one line wrapper for now: we are defaulting the `log` function with an implementation using `debug`, which someone could overwrite. that was my logic